_delphi_utils_python/Makefile

```
@@ -0,0 +1,22 @@
+.PHONY = venv, lint, test, clean
+
+dir = $(shell find ./delphi_* -name __init__.py |  sed 's/\/__init__.py//g;s/\.\///g')
+
+venv:
+	python3 -m venv env
```

```suggestion
	python3.8 -m venv env
```

Necessary because anyone using CMU-standard Ubuntu has 3.6 installed, and thus:
```
$ python3 --version
Python 3.6.9
```